frontend/src/utils/fetches/fetch-courses.ts

```diff
@@ -0,0 +1,18 @@
+import { authenticatedFetch } from "../authenticated-fetch.ts";
+const API_URL = import.meta.env.VITE_APP_API_HOST;
+
+export const fetchCourses = async (user_uid: string) => {
+  try {
+    const response = await authenticatedFetch(
+      `${API_URL}/courses?teacher=${user_uid}`,
```

Comment:
`teacher=` not needed, should be handled by authentication

Reply:
if `teacher=` is gone then all the courses of that user are visible not just the one where it is a teacher of

Reply:
Correct, my BAD